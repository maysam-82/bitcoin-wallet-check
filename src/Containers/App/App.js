import React from 'react';
import Wallet from '../../Components/Wallet/Wallet';
import Bitcoin from '../../Components/Bitcoin/Bitcoin';
import './App.scss';

const App = () => {
	return (
		<div>
			<h2>Money Check</h2>
			<Wallet />
			<Bitcoin />
			<div className="coindesk-container">
				Powered by{' '}
				<a target="_blank" href="https://www.coindesk.com/price/bitcoin">
					CoinDesk
				</a>
			</div>
		</div>
	);
};

export default App;
