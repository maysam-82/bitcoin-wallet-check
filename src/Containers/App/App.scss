.coindesk-container {
	width: 100%;
	font-size: 0.625rem;
	text-align: center;
	a {
		color: #007eff;
		text-decoration: none;
	}
}
