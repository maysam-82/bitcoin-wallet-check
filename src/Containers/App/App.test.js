import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Component', () => {
	const wrapper = shallow(<App />);
	it('should render App properly', () => {
		expect(wrapper.debug()).toMatchSnapshot();
	});
	it('should render `Connect(Wallet)` component', () => {
		expect(wrapper.find('Connect(Wallet)').exists()).toBe(true);
	});
	it('should render `Connect(Bitcoin)` component', () => {
		expect(wrapper.find('Connect(Bitcoin)').exists()).toBe(true);
	});
	it('should contain a link to the coindesk price page', () => {
		expect(wrapper.find('a').props().href).toBe('https://www.coindesk.com/price/bitcoin');
	});
});
