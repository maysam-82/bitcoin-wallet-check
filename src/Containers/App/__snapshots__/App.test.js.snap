// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render App properly 1`] = `
"<div>
  <h2>
    Money Check
  </h2>
  <Connect(Wallet) />
  <Connect(Bitcoin) />
  <div className=\\"coindesk-container\\">
    Powered by
     
    <a target=\\"_blank\\" href=\\"https://www.coindesk.com/price/bitcoin\\">
      CoinDesk
    </a>
  </div>
</div>"
`;
